test/controllers/v1/events_controller_test.rb

```diff
@@ -60,4 +60,19 @@ def teardown
     end
     assert_response :not_found
   end
+
+  test "alert when trying to create an invalid event" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.